src/features/api-keys/routes.ts

```diff
@@ -34,6 +34,8 @@ export const createGenericApiKey: RequestHandler = async (req, res) => {
 
 export const createDeviceApiKey: RequestHandler = async (req, res) => {
 	const deviceId = parseInt(req.params.deviceId, 10);
+	const { name, description }: { name: string; description: string } = req.body;
```

Comment:
You will need to do some validation on `name` and `description` before passing it onwards.

Reply:
@dfunckt What kind of validation are we looking for?

We do similar de-structuring here: https://github.com/balena-io/open-balena-api/pull/948/files#diff-e5d9db05ea54b94163d018aac63bfaaed210c4073f518e0550d97a0a72f2fb1eR102

Reply:
Discussed here: https://www.flowdock.com/app/rulemotion/resin-tech/threads/spWwhMW27JpeCL5byTfh3CJ1N8p